src/core/colorspace_utils.js

```diff
@@ -285,6 +285,11 @@ class ColorSpaceUtils {
   }
 
   static get cmyk() {
+    try {
+      return shadow(this, "cmyk", new CmykICCBasedCS());
+    } catch {
+      warn("CMYK fallback: DeviceCmykCS");
```

Comment:
Nit: Let's match the actual spec-name.
```suggestion
      warn("CMYK fallback: DeviceCMYK");
```